tests/rptest/wasm/native_kafka_producer.py

```diff
@@ -33,6 +36,9 @@ def task_name(self):
         return f"producer-worker-{str(random.randint(0, 9999))}"
 
     def _run(self):
+        producer = KafkaCliTools(self._redpanda)
```

Comment:
I see you've made a different producer in another file , you can delete this file and place it in that other commit